Apps/TimelineApp/feature/user/src/main/java/uk/ac/cam/cares/jps/user/SensorSettingFragment.java

```diff
@@ -1,11 +1,14 @@
 package uk.ac.cam.cares.jps.user;
 
+import static uk.ac.cam.cares.jps.user.LoginFragment.LOGGER;
```

Comment:
should not import the LOGGER from LoginFragment, create one for this class instead